app/controllers/user_controller.rb

```diff
@@ -0,0 +1,29 @@
+class UserController < ApplicationController
+  def index
+    @users = User.all
+    end
+  def add
+    @user = User.find(params[:id])
+  end
+  def create
+    @user = User.new(user[:id])
+     user.save
+    redirect_to @user
+  end
+  def edit
+    @user = User.new(params[:id])
+  end
+  def delete
+    @user = User.find(params[:id])
+  end
+  def destroy
```

Comment:
@IgorKrv why we have the destroy and delete?  what is the difference?

Reply:
delete destroy